packages/flutter/lib/src/painting/binding.dart

```diff
@@ -114,6 +118,41 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
     );
   }
 
+  /// Calls through to [instantiateImageCodecFromBuffer] from [ImageCache].
```

Comment:
The `[instantiateImageCodecFromBuffer]` is missing the "dart:ui." prefix to link to the dart:ui implementation and not itself.

Reply:
Done